### ISSUE TITLE
Teleportation circuit plot

### ISSUE THREAD

**Issue by bettinaheim** [Qiskit/qiskit#991]
### Informations

- **Qiskit version**:
0.19.6

- **Python version**:
3.8.5.

- **Operating system**:
Windows

### What is the current behavior?
The circuit plotted for the attached code is incorrect. 

### Steps to reproduce the problem
Load and plot the circuit.

### What is the expected behavior?
Getting a standard teleport circuit diagram. 

### Suggested solutions
None

Code to reproduce: 

OPENQASM 2.0;
include "qelib1.inc";

qreg qubits[3];
creg corrz[1];
creg corrx[1];
creg final[1];

h qubits[0];

h qubits[1];
cx qubits[1], qubits[2];

cx qubits[0], qubits[1];
h qubits[0];

measure qubits[0] -> corrz[0]; 
measure qubits[1] -> corrx[0];

if (corrz==1) z qubits[2]; 
if (corrx==1) x qubits[2];

h qubits[2];

measure qubits[2] -> final[0];

**Comment by RafeyIqbalRahman** [comment]
These are the measurement probabilities I got after running your code on the official IBM Quantum Experience Portal.

![bug](https://user-images.githubusercontent.com/59226057/89397871-d4150600-d729-11ea-9f84-21b23cab1175.PNG)

The measurement probabilities seem right for teleportation.

**Comment by bettinaheim** [comment]
@RafeyIqbalRahman Yes, indeed - sorry for having been unclear: the piece that seems incorrect is the emitted circuit diagram (the classical control is shown before the measurement that sets it).
